addons/binding/org.openhab.binding.solaredge/src/main/java/org/openhab/binding/solaredge/internal/model/AggregateDataResponse.java

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.solaredge.internal.model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * this class is used to map the aggregate data json response
+ *
+ * @author Alexander Friese - initial contribution
+ *
+ */
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class AggregateDataResponse implements DataResponse {
+
+    private static final String UNIT_WH = "Wh";
+    private static final String UNIT_KWH = "KWh";
+
+    @JsonIgnoreProperties(ignoreUnknown = true)
+    public static class Value {
+        public String value;
```

Comment:
Looking at the example json below it looks like value is a double and not a string. Why have it specified as a String here?

Reply:
this was copied from another class where the result could actually be a String, too. But in this case it can only be Double so I changed it